include/fastdds/rtps/transport/UDPv4TransportDescriptor.h

```diff
@@ -31,15 +31,18 @@ namespace rtps{
  * - interfaceWhiteList: Lists the allowed interfaces.
  * @ingroup TRANSPORT_MODULE
  */
-typedef struct UDPv4TransportDescriptor: public UDPTransportDescriptor
+typedef struct UDPv4TransportDescriptor : public UDPTransportDescriptor
```

Comment:
Avoid typedef trick.
Check class documentation, as it seems to only mention fields on SocketTransportDescriptor